index.js

```diff
@@ -139,6 +140,24 @@ function Hook (modules, options, onrequire) {
       return self._origRequire.apply(this, arguments)
     }
 
+    return patchedRequire.call(this, arguments, false)
+  }
+
+  this._getBuiltinModule = process.getBuiltinModule = function (id) {
```

Comment:
I don't think we should be setting `process.getBuiltinModule` in versions of Node.js that don't have it.
Thoughts?

Reply:
Fixed in eb50f6d 👍